src/main/java/build/buildfarm/worker/cgroup/Group.java

```diff
@@ -44,22 +55,67 @@ public final class Group {
   @Getter private final Cpu cpu;
   @Getter private final Mem mem;
 
-  @SuppressWarnings("NullableProblems")
-  private Group(String name, Group parent) {
+  @SuppressWarnings(
+      "PMD.MutableStaticState") // Unit tests set this. When CGroups v1 support is gone, this will
+  // go away, too.
+  protected static CGroupVersion VERSION = discoverCgroupVersion();
+
+  private static CGroupVersion discoverCgroupVersion() {
+    /* Try to figure out which version of CGroups is available. */
+    try {
+      List<String> allMounts = Files.readAllLines(Paths.get("/proc/mounts"));
```

Comment:
This [stackexchange post](https://unix.stackexchange.com/questions/471476/how-do-i-check-cgroup-v2-is-installed-on-my-machine) highlights what I think is a better way forward here.  The java equivalent of `stat -fc %T /sys/fs/cgroup/` and splitting on tmpfs/cgroup is the tl;dr.

There are a lot of things I'd prefer to see made constants here regardless